packages/core/stories/toolkit-provider.doc.stories.mdx

```diff
@@ -96,6 +96,129 @@ It requires no configuration, but is included in your app when you use a Toolkit
 AriaAnnouncer is ever required, only the outermost instance of ToolkitProvider will instantiate the AriaAnnouncer. If you use
 nested ToolkitProviders, these will not inject further instances of AriaAnnouncer.
 
+## Breakpoints
+
+The `breakpoints` prop on ToolkitProvider can be used to customize the css media queries that custom hooks like `useBreakpoints` and useCurrentBreakpoint` are using internally. These hooks are usually used to enable the responsive props, where you can pass multiple values (one per breakpoint) or a single value that would take effect across all different screen sizes. These props can mostly be found in the layout components for now. Look for an example in the [FlexLayout](/docs/documentation-core-layout-flexlayout--page) component documentation.
```

Comment:
Missing \` in front of `useCurrentBreakpoint`

Reply:
👍